tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -585,7 +596,7 @@ function setupStorybook(tree: Tree, options: NormalizedSchema) {
 
   const template = {
     projectReferences: { path: './.storybook/tsconfig.json' },
-    exclude: ['**/*.stories.ts', '**/*.stories.tsx'],
+    exclude: ['**/*.stories.ts', '**/*.stories.tsx', '**/*.cy.ts', '**/*.cy.tsx'],
```

Comment:
exclude processing cypress files within `tsconfig.lib.json`

Reply:
**chore:** this is breaking separation of concerns. we need create separate function for setting up cypress that will handle these things.

Reply:
update: we already have it, lets move the logic to `setupCypress`

Reply:
[21ba097](https://github.com/microsoft/fluentui/pull/24458/commits/21ba09778dcb1ac097d1752f443781ed08beec78)